scRNA-seq-advanced/03-dataset_integration.Rmd

```diff
@@ -505,8 +496,9 @@ reducedDim(merged_sce, "merged_PCA") <- merged_pca[[1]]
 ```
 
 
-Now that we have the PCA matrix, we can proceed to calculate UMAP for visualization. 
-We'll calculate this as "usual":
+Now that we have the PCA matrix, we can proceed to calculate UMAP to visualize the uncorrected merged data.
+
+We'll calculate UMAP as "usual":
 ```{r merged_umap}
 # add merged_UMAP from merged_PCA
 merged_sce <- scater::runUMAP(merged_sce,
```

Comment:
It might be worth adding a small comment here that we are explicitly providing the `name` argument, I don't think we've seen this before yet?